src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -348,20 +347,20 @@ private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] ar
             {
                 throw new RpcException(-100, "Unknown contract");
             }
-            var methodName = "verify";
+            var md = contract.Manifest.Abi.GetMethod("verify", 0);
```

Comment:
Can we set `pcount` to -1 here in order to be able to call `verify` with arguments? Just like in https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/Helper.cs#L222.

Reply:
Updated it.